public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -67,22 +89,40 @@ function verify_credentials(){
         $meta_key = 'event_users';
         $event_users = get_post_meta( $event_id, $meta_key, true );
         $rsvp_date = date("d/m/y");
-        $rsvp_guest_id = $wpdb->get_var("SELECT post_id FROM $tablename WHERE meta_value = '".$_POST['guest_email']."'");
+        $guest_email = $_POST['guest_email'];
+        $rsvp_guest_id = $wpdb->get_var("SELECT post_id FROM $tablename WHERE meta_value = '".$guest_email."'");
         $hash = get_post_meta($rsvp_guest_id,'password',true);
-
+        $guest_name = get_the_title($rsvp_guest_id);
+        $soiree_name = get_the_title($event_id);
+        $soiree_date = get_post_meta($event_id,'event_date',true);
+        $host_info = get_userdata(get_current_user_id());
+        $host_name = $host_info->first_name;
+        $host_email = get_option('admin_email');
```

Comment:
Are we using this variable? Check line [114](https://github.com/Shubham2301/phase2/pull/33/files#diff-01391f7f399cd5492a25445b0fd9ec87R114).